src/components/UI/input/input.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './input.css'
+
+function isInvalid({valid, touched, shouldValidate}) {
+    return !valid && shouldValidate && touched
+}
+
+
+const Input = props => {
+
+    const inputType = props.type || 'text'
+    const cls = ['Input']
+    const htmlFor = `${inputType}-${Math.random()}`
+
+    if (isInvalid(props)) {
+        cls.push('invalid')
```

Comment:
not added 'invalid' className, because when "isInvalid" function get props from src/containers/QuizCreator/QuizCreator.js line 78, 79, 80 verification is not performed,

Reply:
The properties here are correct.